package.json

```diff
@@ -44,7 +44,8 @@
     "test-python-scripts": "npm run pep8 && PYTHONPATH=./script python -m unittest discover -s ./script/test",
     "update_symlink": "node ./build/commands/scripts/commands.js update_symlink",
     "build_fuzzer": "node ./build/commands/scripts/commands.js build_fuzzer",
-    "fuzzer": "node ./build/commands/scripts/commands.js run_fuzzer"
+    "fuzzer": "node ./build/commands/scripts/commands.js run_fuzzer",
+    "perf_tests": "node ./build/commands/scripts/commands.js run_perf_tests"
```

Comment:
why `perf_tests` but `run_perf_tests`? makes no sense.

I think `fuzzer` -> `run_fuzzer` is the one incorrect here and should be fixed. cc @boocmp

Reply:
Then command will looks like ```npm run run_fuzzer <target>```.

Reply:
actually, I agree. Otherwise we will have `npm run run_...`. Let's keep it as is.